Frameworks/UIKit/UILabel.mm

```diff
@@ -193,15 +209,29 @@ - (instancetype)initWithCoder:(NSCoder*)coder {
         _font = font;
 
         _alignment = (UITextAlignment)[coder decodeInt32ForKey:@"UITextAlignment"];
-        _adjustFontSize = [coder decodeInt32ForKey:@"UIAdjustsFontSizeToFit"];
 
         if ([coder containsValueForKey:@"UINumberOfLines"]) {
             _numberOfLines = [coder decodeInt32ForKey:@"UINumberOfLines"];
         } else {
             _numberOfLines = 1;
         }
 
-        _minimumFontSize = [coder decodeFloatForKey:@"UIMinimumFontSize"];
+        if ([coder containsValueForKey:@"UIMinimumScaleFactor"]) {
+            _minimumScaleFactor = [coder decodeFloatForKey:@"UIMinimumScaleFactor"];
+            _minimumScaleFactorHasBeenSet = YES;
+            _adjustFontSize = YES;
+        } else if ([coder containsValueForKey:@"UIMinimumFontSize"]) {
+            _minimumFontSize = [coder decodeFloatForKey:@"UIMinimumFontSize"];
+            _adjustFontSize = YES;
+        }  else if ([coder containsValueForKey:@"UIAdjustsFontSizeToFit"]) {
+            _adjustFontSize = [coder decodeInt32ForKey:@"UIAdjustsFontSizeToFit"];
+            assert(_adjustFontSize == NO);
+        } else {
+            _adjustFontSize = NO;
+        }
+
+        if (_adjustFontSize) {
+        }
```

Comment:
weird, some change is missing from my CR.

---
In reply to: [99672319](https://github.com/Microsoft/WinObjC/pull/1918#discussion_r99672319) [](ancestors = 99672319)